### PR TITLE
Feature/random stuff

### PR DESCRIPTION
**Pull request by martinstannard**
Cleanups up elm file structure by separating in to different files, start on GameOver functionality

### CODE REVIEW

**Comment by martinstannard** [comment]
@gogogarrett @Bockit

**Comment by martinstannard** [comment]
I've deployed this to lettersmash.heroku.com

File: apps/library/lib/library/game.ex [hunk 1]
```diff
@@ -112,7 +120,8 @@ defmodule Library.Game do
       %{
         game_state |
           board: Board.generate,
-          word_list: []
+          wordlist: [],
+          game_over: false
       }
     }
```
Comment (gogogarrett) [inline apps/library/lib/library/game.ex]: Just a though, What if we define "initalState" as a module attr then just reference that here - kind of how we do in elm

Comment (martinstannard) [inline apps/library/lib/library/game.ex]: Not sure I get what you mean?

**Pull request merged**